{{ cookiecutter.__dirname }}/.github/workflows/build.yml

```diff
@@ -54,6 +54,26 @@ jobs:
         pre-commit run --all-files
     - name: Build assets
       run: circuitpython-build-bundles --filename_prefix {% raw %}${{ steps.repo-name.outputs.repo-name }}{% endraw -%} {% if cookiecutter.target_bundle == 'Community' %} --package_folder_prefix {% if cookiecutter.library_prefix -%} {{ cookiecutter.library_prefix | lower | replace(' ', '_') }}_ {%- else %}""{%- endif -%}{%- endif %} --library_location .
+    - name: Clone Size Tools
+      uses: actions/checkout@v3
+      with:
+        repository: 'circuitpython/CircuitPython_Org_size_tools'
+        path: 'CircuitPython_Org_size_tools'
+    - name: Check Sizes
+      run: |
+        python CircuitPython_Org_size_tools/size_tools.py
+    - name: Check sizes.txt file existence
+      id: check_sizes_file
+      uses: andstor/file-existence-action@v1
+      with:
+        files: "sizes.txt"
+    - name: Size Measurement Comment
+      if: github.event_name == 'pull_request' && steps.check_sizes_file.outputs.files_exists == 'true'
+      uses: machine-learning-apps/pr-comment@master
+      env:
+        GITHUB_TOKEN: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
```

Comment:
Latest commit removes the extra space here. This was copied from another similar usage of the token which has now been removed by #202